scripts/README.md

```diff
@@ -29,17 +42,8 @@ The following steps must be proposed as a pull request.
 
 ### Documentation
 
-Push the next branch on the release branch to deploy the documentation with the latest changes. It lives at https://material-ui.netlify.app/. Force push if necessary.
```

Comment:
If I understand this correctly, only `yarn docs:deploy` is needed to do this step.

The way that this was written originally made me think "Push the next branch..." and and `yarn docs:deploy` were two separate actions